internal/vds/metadatahandle.cpp

```diff
@@ -81,10 +81,15 @@ std::string MetadataHandle::crs() const noexcept (true) {
 }
 
 std::string MetadataHandle::format() const noexcept (false) {
-    auto format = this->m_layout->GetChannelFormat(0);
+    /*
+     * We always request data in OpenVDS::VolumeDataFormat::Format_R32 format
+     * as this seems to be intended way when working with openvds.
+     * Thus users will always get data returned as f4.
+     *
+     * We also assume that server code is run on a little-endian machine.
```

Comment:
I would love a link to the issue on gitlab where this is explained in more detail